docs/static/releasenotes.asciidoc

```diff
@@ -53,6 +54,44 @@ This section summarizes the changes in the following releases:
 * <<logstash-7-0-0-alpha2,Logstash 7.0.0-alpha2>>
 * <<logstash-7-0-0-alpha1,Logstash 7.0.0-alpha1>>
 
+[[logstash-7-15-2]]
+=== Logstash 7.15.2 Release Notes
+
+
+[[notable-7-15-2]]
+==== Performance improvements and notable issues fixed
+
+* Upgrade jruby-openssl to 0.11.0 to properly support alt-chain certificate verifications in plugins using the Ruby
+  net/http stack.
+  The recent Let's Encrypt "DST Root CA X3" expiration has caused issues in Logstash plugins such as S3.
+  We're including the upstream fix to avoid these issues and support alternate chain TLS certificate verification.
```

Comment:
```suggestion
**Jruby-openssl upgrade.** We've upgraded {ls} to 0.11.0 to properly support
alt-chain certificate verifications in plugins using the Ruby net/http stack.
The recent Let's Encrypt "DST Root CA X3" expiration has caused issues in
Logstash plugins such as S3.
We're including the upstream fix to avoid these issues and support alternate
chain TLS certificate verification.
```

Reply:
The way you have it formatted is what we typically do. In this case, the single bullet under a major heading looks like a mistake.  I've suggested an alternate to compare/contrast to the current formatting (as seen in the preview: https://logstash_13390.docs-preview.app.elstc.co/guide/en/logstash/7.15/logstash-7-15-2.html). 

If you chose to go with suggested formatting, it would look like this: 
<img width="861" alt="Screen Shot 2021-11-08 at 5 13 44 PM" src="https://user-images.githubusercontent.com/35154725/140827385-cef54696-8dea-45f9-8c3b-be1868413a68.png">